zebra-chain/src/amount.rs

```diff
@@ -311,6 +342,9 @@ pub enum Error {
     MultiplicationOverflow { amount: i64, multiplier: u64 },
     /// cannot divide amount {amount} by zero
     DivideByZero { amount: i64 },
+
+    /// Attempt to sum with overflow
```

Comment:
Can we try to match the wording of the other overflow error?
(These comments are used as error messages by the `thiserror` derive.)

```suggestion
    /// i64 overflow when summing i64 amounts
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/2500/commits/81b4e43235fa192183896ec7e0d90cff27760929